internal/services/consumer/consumer_test.go

```diff
@@ -0,0 +1,312 @@
+package consumer
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/DIMO-Network/shared"
+	"github.com/DIMO-Network/trips-api/internal/services/pg"
+	"github.com/DIMO-Network/trips-api/internal/test"
+	"github.com/DIMO-Network/trips-api/models"
+	"github.com/rs/zerolog"
+	"github.com/segmentio/ksuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/volatiletech/null/v8"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+)
+
+var (
+	migrationsDirRelPath = "../../../migrations"
+)
+
+var createDevice shared.CloudEvent[UserDeviceMintEvent] = shared.CloudEvent[UserDeviceMintEvent]{
+	Type: UserDeviceMintEventType,
+	Data: UserDeviceMintEvent{
+		Device: struct {
+			ID string "json:\"id\""
```

Comment:
If you use backticks you do not have to escape.
```suggestion
			ID string `json:"id"`
```